src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  const miniDiscount = 20;
+  const basicDiscount = 50;
+
+  let totalAmount = 0;
+
+  if (days < 3) {
+    totalAmount = days * dayPrice;
+  } else if (days >= 3 && days < 7) {
```

Comment:
it is not necessary  to use `days >=3` as we checked whether the number is smaller than 3 in first condition, so this condition is already checking number from 3 to 7 without `days >=3`

Reply:
check @annholovko comment

Reply:
good job!🔥
left one suggestion